app/controllers/master/api/providers_controller.rb

```diff
@@ -187,16 +187,16 @@ def plans
   end
 
   def get_partner
-    @partner = Partner.find_by_system_name(params[:partner])
+    @partner = Partner.find_by(system_name: params.permit(:partner)[:partner])
```

Comment:
Do we need `permit`? `Partner.find_by(system_name: params[:partner])` should be fine as far as I can tell.